modules/component-store/src/component-store.ts

```diff
@@ -36,6 +36,16 @@ export const INITIAL_STATE_TOKEN = new InjectionToken(
   '@ngrx/component-store Initial State'
 );
 
+export type SelectorResults<Selectors> = {
```

Comment:
```suggestion
export type SelectorResults<Selectors extends Observable<unknown>[]> = {
```
right?

Reply:
Ah yes, that makes a lot more sense (as do the other comments), thanks for the review!